docs/src/main/resources/jasper-reports-part.html

```diff
@@ -95,7 +95,7 @@
 <p>One of the most important change from version 2 to 3 of Mapfish Print, at 
 least for users, 
 is the use of jasper template (jrxml) as basis of layout. The chapter 
-“General workflow for adapting existing templates”
+"General workflow for adapting existing templates"
```

Comment:
I think non-Ascii quote should be entered with html entities : 	&ldquo; and &rdquo;. 
Personnaly, I would prefer traditional ascii quote !